jlm/llvm/opt/alias-analyses/PointerObjectSet.hpp

```diff
@@ -140,8 +185,8 @@ class PointerObjectSet final
   // Mapping from global variables declared with delta nodes to PointerObject
   std::unordered_map<const delta::node *, PointerObject::Index> GlobalMap_;
 
-  // Mapping from functions declared with lambda nodes to PointerObject
-  std::unordered_map<const lambda::node *, PointerObject::Index> FunctionMap_;
+  // Bidirectional mapping from functions declared with lambda nodes to PointerObject
```

Comment:
The comment is unnecessary in my opinion. The code states exactly what you say in the comment here.